test/AutoRest.TestServer.Tests/lro.cs

```diff
@@ -101,20 +101,6 @@ public class LroTest: TestServerTestBase
             Assert.AreEqual("Succeeded", result.Value.ProvisioningState);
         });
 
-        [Test]
-        public Task LROPatchInlineCompleteIgnoreHeaders() => Test(async (host, pipeline) =>
-        {
-            var operation = await new LROsClient(ClientDiagnostics, pipeline, host).StartPatch200SucceededIgnoreHeadersAsync();
-            var resp = await operation.WaitForCompletionAsync().ConfigureAwait(false);
-        });
-
-        [Test]
-        public Task LROPatchInlineCompleteIgnoreHeaders_Sync() => Test((host, pipeline) =>
-        {
-            var operation = new LROsClient(ClientDiagnostics, pipeline, host).StartPatch200SucceededIgnoreHeaders();
-            var resp = WaitForCompletion(operation);
-        });
```

Comment:
Why are we removing these tests?

Reply:
Thanks. I'm about to ask why the generated codes are so different :)